tests/src/test/scala/org/http4s/EntityEncoderSpec.scala

```diff
@@ -127,4 +120,19 @@ class EntityEncoderSpec extends Http4sSpec {
       EntityEncoder[IO, ModelB] must_== w2
     }
   }
+
+  {
+    import org.scalacheck._, Arbitrary.arbitrary
+    implicit def arbitraryIO[A](implicit A: Arbitrary[A]): Arbitrary[IO[A]] =
```

Comment:
A better version of this exists in cats-effect, but isn't published.  I'm asking about this.